DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Parts/TitlePartGraphSyncer.cs

```diff
@@ -34,5 +43,23 @@ public override Task AddSyncComponents(JObject content, IGraphMergeContext conte
                 NodeTitlePropertyName,
                 context.NodeWithOutgoingRelationships.SourceNode));
         }
+
+        //we're now doing this for all cloned content items, we happy with that?
```

Comment:
We _could_ limit it to pages, but it probably makes sense for all content types.
Whatcha think?

Reply:
Yeah all content types is fine by me!